src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+
+export enum TextError {
+  Add = 'Unable to add a todo',
+  Delete = 'Unable to delete a todo',
+  Update = 'Unable to update a todo',
+  Title = 'Please add valid title',
+  Data = 'Unable to load data',
+  noUser = 'No user found',
+}
+
+type Props = {
+  error: TextError;
+  setError: (errorMessage: TextError | null) => void;
+};
+
+export const ErrorNotification: React.FC<Props> = ({ error, setError }) => {
+  const [errorText] = useState(TextError.Add);
+
+  const closeError = () => {
```

Comment:
Do it with timer, so the user could see an error=)

Reply:
I did timer for autoclosing function, but here is handle closing - I believe it should be without timer?

Reply:
You need to create setTimeout on the first render and clear timeout on unmount(not inside of this function).